recipes/comet-ms/build.sh

```diff
@@ -2,19 +2,12 @@
 set -e
 set -x
 
-platform="$(uname)"
-unzip comet_source_"$PKG_VERSION".zip
-if [ "$platform" = "Darwin" ]; then
-    sed -i.bak -e 's/ -static//' -e 's/ -o / -headerpad_max_install_names&/' Makefile
-    sed -i.bak -e "s/sha1Report='..'/sha1Report=NULL/" MSToolkit/include/MSReader.h
-    sed -i.bak "s/ off64_t / off_t /g" CometSearch/Common.h
-    sed -i.bak "s/fseeko64/fseeko/g;s/ftello64/ftello/g" MSToolkit/include/mzParser.h
-    sed -i.bak "s/fseeko64/fseeko/g;s/ftello64/ftello/g" CometSearch/Common.h
-    sed -i.bak "s/pthread_yield/sched_yield/g;" CometSearch/ThreadPool.h
-fi
-sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" CometSearch/Makefile
+export INCLUDE_PATH="${PREFIX}/MSToolkit/include"
+export CPPFLAGS="-I${PREFIX}/MSToolkit/include"
+
 sed -i.bak "s#gcc#${CC}#;s#g++#${CXX}#" MSToolkit/Makefile
-make CXX=${CXX} CXXFLAGS="${CXXFLAGS} -mcmodel=large"
+make CXX=${CXX} CXXFLAGS="${CXXFLAGS} -mcmodel=large -fPIC -I./include" CFLAGS="${CFLAGS} -static -mcmodel=large -fPIC -I./include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -DGCC -DHAVE_EXPAT_CONFIG_H"
```

Comment:
```suggestion
make CXX=${CXX}
```

Reply:
Let's try without messing with the flags. I think for a standalone executable it might be fine ignoring conda specifics.